ruskeyverbs/templates/ruskeyverbs/list_view_per_user.html

```diff
@@ -0,0 +1,43 @@
+{% extends "ruskeyverbs/base_generic.html" %}
+
+{% block extra_head_content %}
+  {% load static %}
+  <link rel="stylesheet" href="{% static "ruskeyverbs/css/verb_list_style.css" %}"/>
+
+{% endblock %}
+
+{% block content %}
+  <h1 class='text-center'>Verbs</h1>
+
+  {% if verb_list %}
+  {% load ruskeyverbs_extras %}
+  <div class="container">
+  {% for verb in verb_list %}
+  {% if forloop.first %}
+  <div class="row">
+  {% endif %}
+    <div class="col-sm-4">
+      <a href="{% url 'verb-details' verb.pk %}">
+        <div class="verb_block col-one-link">
+          <h1 class='text-center'>{{ verb.pk }}</h1>
+          <h2 class='text-center'>{{ verb.infinitive }}</h2>
+          <p class='text-center'>
+            {{ verb.meaning|truncatechars:50 }}
+          </p>
+          <p class='text-center'>
+            due:
+            <span class="{% if verb|overdue:current_user %}font-weight-bold font-italic text-danger{% endif %}">
+              {{ verb|duedate:current_user }}
+            </span>
+          </p>
+        </div>
+      </a>
+    </div>
+    {% if forloop.counter|divisibleby:3 %}</div><div class="row">{% endif %}
```

Comment:
What is your actual intention for display on this page? Is it just that there should there always be 3 lists no matter the window size? Or is there more to it? If it's just wanting things to be in three columns then you can almost certainly achieve that with just bootstrap and no python code at all...

Reply:
The whole intention is to have the items displayed in three rows if the screen is sufficiently large. I guess the only advantage I saw in using the template logic here was that each row would have <row> and </row> tags. However, it seems like just adding all the divs with the "col-sm-4" class to a single row achieves the same effect.